.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!--
+Welcome to the Jaeger project! 👋🎉
+
+- Please search for existing issues to avoid creating duplicate bugs/feature requests.
+- Please be respectful and considerate of others when commenting on issues.
+- Please provide as much information as possible so we all understand the issue.
+- If you only have a question, you may get a faster response by asking in
```

Comment:
Something mentioned this morning about not spamming multiple channels. The grpc template has this:
```
*Please don't double post your questions in more locations; we are monitoring both channels, and the time spent de-duplicating questions is better spent answering more user questions.*
```
might be worth enforcing here?

Reply:
added below, but in a much shorter form - I feel that too many instructions and people will simply skip the whole thing